vnext/Desktop/WebSocket.cpp

```diff
@@ -402,34 +402,34 @@ void BaseWebSocket<Protocol, SocketLayer, Stream, Resolver>::Ping() {
 #pragma region Handler setters
 
 template <typename Protocol, typename SocketLayer, typename Stream, typename Resolver>
-void BaseWebSocket<Protocol, SocketLayer, Stream, Resolver>::SetOnConnect(function<void()> &&handler) {
+void BaseWebSocket<Protocol, SocketLayer, Stream, Resolver>::SetOnConnect(function<void __cdecl()> &&handler) {
   m_connectHandler = handler;
 }
 
 template <typename Protocol, typename SocketLayer, typename Stream, typename Resolver>
-void BaseWebSocket<Protocol, SocketLayer, Stream, Resolver>::SetOnPing(function<void()> &&handler) {
+void BaseWebSocket<Protocol, SocketLayer, Stream, Resolver>::SetOnPing(function<void __cdecl()> &&handler) {
```

Comment:
>__cdecl [](start = 85, length = 7)

hmm, is this needed? how does std::function deal with calling convention?

Reply:
I do not know it for a fact, but suspect that a consuming application built with `__stdcall` may have the functors act as `void __stdcall` functions. With this change, IWebSocket should expect functors that are equivalent to `void __cdecl`.

Am I making a wrong assumption here?
In any case, that change should be innocuous and could be reverted if I'm proven wrong.

Reply:
Confirmed. If a lambda is provided without expressing calling convention, it will go with the compiler's default.
It's best to require __cdecl explicitly.